lib/utils.pm

```diff
@@ -681,7 +681,7 @@ sub validate_repos_sle {
             validatelr({product => "SLE-", uri => "$cd:///", version => $version});
         }
         elsif (check_var('FLAVOR', 'Server-DVD-HA')) {
-            validatelr({product => "SLES", uri => "$cd:///", version => $version});
+            validatelr({product => "SLES", uri => check_var('ARCH', 's390x') ? "ftp://" : "$cd:///", version => $version});
```

Comment:
L655 already sets the $cd parameter based on other parameters. I suggest to do the same for s390x also there